app/views/finances/_form.html.erb

```diff
@@ -1,52 +1,248 @@
+    }
+    .imsa-logo {
+      position: fixed;
+      top: 25px;
+      left: 50px;
+      width: 100px;
+    }
+    .profile-icon {
+      position: fixed;
+      top: 25px;
+      right: 50px; 
+      width: 100px; 
+      cursor: pointer;
+    }
 
-      <ul>
-        <% finance.errors.each do |error| %>
-          <li><%= error.full_message %></li>
-        <% end %>
-      </ul>
+    h2 {
+      color: #333;
+    }
+
+    p {
+      color: #666;åå
+    }
+
+    a {
+      color: #007bff;
+      text-decoration: none;
+      margin: 5px;
+    }
+
+    .admin-links,
+    .user-links {
+      display: block; 
+    }
+
+    .user-links,
+    .admin-info {
+      display: block; 
+    }
+
+    .toggle-links-button {
+      padding: 10px;
+      margin: 10px;
+      cursor: pointer;
+    }
+
+    ul {
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    li {
+      display: inline;
+      margin-right: 10px;
+    }
+
+    nav {
+      background-color: #007bff;
+      padding: 5px; 
+    }
+
+    nav a {
+      color: #fff;
+    }
+
+    .content {
+      margin-top: 60px; 
+    }
+
+    .navbar {
+      background-color: #007bff;
+      padding: 5px;
+      width: 55%;
+      margin: auto; 
+      margin-top: 5vh;
+      display: flex;
+      align-items: center; 
+      justify-content: space-evenly;
+      z-index: 1000;
+    }
+
+    .navbar a {
+      color: #fff;
+    }
+
+    .navbar .active a {
+      background-color: #C0C0C0; 
+      color: #007bff; 
+      border-radius: 5px; 
+    }
+
+    .navbar-nav {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+    }
+
+    .member-navbar {
+      width: 30%;
+    }
+    @import url('https://fonts.googleapis.com/css2?family=Lateef&display=swap');
+
+
+    .side-header {
+      transform: rotate(270deg);
+      position: fixed;
+      top: 75%; 
+      left: 11%; 
+      transform-origin: bottom left; 
+      letter-spacing: 0.8em; 
+      font-size: 100px; 
+      font-family: 'Mustopha'; 
+    }
+
+
+    .form-formatting { 
+      align-items: center; 
+    }
+
+  </style>
+</head>
+  
+<div class="form-formatting">
+  <%= form_with(model: finance) do |form| %>
+    <% if finance.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(finance.errors.count, "error") %> prohibited this finance from being saved:</h2>
+
+        <ul>
+          <% finance.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <br>
+
+    <div>
+      <%= form.label :email, style: "display: block; margin-bottom: 10px;" %>
+      <%= form.text_field :email, readonly: true, style: "margin-bottom: 10px;" %>
+    </div>
+
+    <div>
+      <%= form.label :eventTitle, "Event Title", style: "display: block; margin-bottom: 10px;" %>
+      <% if current_member&.position == 'President' || current_member&.position == 'Finance Officer' %>

```

Comment:
This code where the form changes depending on who submitted it is very useful. We will also need to let the customer know that "Finance Officer" is the official title. May need to be changed.